docs/howto/setup_and_run_vertexing.md

```diff
@@ -0,0 +1,130 @@
+# ACTS Vertexing Tutorial - Example: Adaptive Multi-Vertex Finder (AMVF) - Pythia8
+This tutorial sets up and runs the ACTS Adaptive Multi-Vertex Finder on smeared ACTS Pythia8 truth tracks.
+*Note*: You have to have Pythia8 available on your machine for following this tutorial.
+## Prerequisites
+For setting up and building ACTS, please refer to the general how-to ACTS guide. For this example you will need to enable Pythia8 in the examples by adding
+```bash
+-DACTS_BUILD_EXAMPLES_PYTHIA8=ON
+```
+to your cmake command.
+
+Note: Additionally adding
+```bash
+-DCMAKE_BUILD_TYPE=Release
+```
+to your cmake command will significantly speed up the execution time of the vertexing example algorithm below, it will however slow down the compilation process.
+
+## Setting up an ACTS Adaptive Multi-Vertex Finder Algorithm
+
+A template algorithm file with an (almost) empty ```execute()``` method to be filled in the following is provided here:
+```bash
+../Examples/Algorithms/Vertexing/src/ExampleAMVFAlgorithm.cpp
+```
+Open the file in your editor and let's start setting up the AMVF. We will start setting up all necessary components in the ```execute()``` method.
+*Note:* You would normally **not** want to do all the following setup steps in the ```execute()``` method (that is run on every single event), but rather in some kind of initialization method that is executed only once (which is however not available in the lightweight example framework we will be using here).
```

Comment:
Nitpicking: this kind of initialization should be done in the constructor. Saying this functionality is not available is misleading.

Reply:
Updated.